### ISSUE TITLE
Pixi.js missing default export

### ISSUE THREAD

**Issue by scythianfuego** [Nazariglez/pixi-keyboard#2]
As the default export will not exist in future according to the pixijs/pixi.js#3204 , could you please fix imports inside KeyboardManager.js and index.js?

```
// pre-4.1.0
import PIXI from "pixi.js";

// since 4.1.0
import * as PIXI from "pixi.js";
```

**Comment by Nazariglez** [comment]
Hi @scythianfuego, i will change this in this days, thanks!

**Comment by Nazariglez** [comment]
Fixed by #4
